msg/vehicle_local_position.msg

```diff
@@ -26,16 +26,16 @@ float32 z_deriv				# Down position time derivative in NED earth-fixed frame, (me
 # Velocity reset delta
 float32[2] delta_vxy
 uint8 vxy_reset_counter
-
 float32 delta_vz
 uint8 vz_reset_counter
+
 # Acceleration in NED frame
-float32 ax        # North velocity derivative in NED earth-fixed frame, (metres/sec^2)
-float32 ay        # East velocity derivative in NED earth-fixed frame, (metres/sec^2)
-float32 az        # Down velocity derivative in NED earth-fixed frame, (metres/sec^2)
+float32 ax        			# North velocity derivative in NED earth-fixed frame, (metres/sec^2)
+float32 ay        			# East velocity derivative in NED earth-fixed frame, (metres/sec^2)
+float32 az        			# Down velocity derivative in NED earth-fixed frame, (metres/sec^2)
```

Comment:
Nothing is using accelerations right now (I think). Should remove it if it's not used.

Reply:
If that's everyone's consideration I can remove it yes.

Reply:
I expect this to be commonly used, so let's keep it until we understand our VIO pipelines better.